### PR TITLE
Add needs picture desk

### PR DESCRIPTION
**Pull request by aug24**
## What does this change?
Adds a flag column for picture desk status.  Needs/Approved, exactly as per Legal.
Similarly, adds edit capability for that flag in the management pane.

## How to test
CODE

## How can we measure success?
Can set and view icons 

## Have we considered potential risks?
Yes.

### CODE REVIEW

File: .java-version [hunk 1]
```diff
@@ -1 +1 @@
-1.8.0.232
+1.8.0.242
```
Comment (jennygrahamjones) [inline .java-version:1]: Was there a particular issue that prompted this?

File: public/components/content-list-item/templates/needsPictureDesk.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<td class="content-list-item__field--needs-picture-desk" title="Needs Picture Desk: {{ contentItem.needsPictureDesk.toLowerCase() }}">
+
+    <i ng-if="contentItem.needsPictureDesk == 'REQUIRED' || contentItem.needsPictureDesk == 'COMPLETE'" class="content-list-item__icon--incopy content-list-item__icon--inline" wf-icon="needs-picture-desk"></i>
```
Comment (jennygrahamjones) [inline public/components/content-list-item/templates/needsPictureDesk.html:3]: Not strictly related to this PR, but is it worth switching out `content-list-item__icon--incopy` for something like `content-list-item__icon--needs...`? At the minute, both this and `needsLegal` are using styling that seems to be tied to another icon.

File: .java-version (hunk 1 above)
Comment (aug24) [reply .java-version]: No, it's a mistake.  Reverted.

File: public/components/content-list-item/templates/needsPictureDesk.html (hunk 2 above)
Comment (aug24) [reply public/components/content-list-item/templates/needsPictureDesk.html]: Done

**Review by jennygrahamjones — APPROVED** [review]

**Pull request merged**